axlearn/common/attention.py

```diff
@@ -2801,6 +2805,7 @@ def _linear2(x):
         self._add_tensor_stats("inputs", inputs)
 
         remat_pt2 = "linear2"
+        remat_pt3 = "feed_forward_output"
```

Comment:
hmm, is this necessary? Seems quite wasteful to checkpoint an activation after a norm? I would recommend restraining from making change here.